doc/swagger.json

```diff
@@ -5,6 +5,71 @@
     "version": "0.1"
   },
   "paths": {
+    "/taoTestCenter/api/eligibilities": {
+      "get": {
+        "tags": [
+          "eligibilities"
+        ],
+        "summary": "Search for eligibilities",
+        "description": "Search for eligibilities",
+        "operationId": "oat\\taoTestCenter\\controller\\RestEligibilities::get",
+        "parameters": [
+          {
+            "name": "delivery",
+            "in": "query",
+            "description": "Delivery Uri (Url encoded)",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "testCenter",
+            "in": "query",
+            "description": "Test center Uri (Url encoded)",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Eligibility data",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "example": {
+                    "delivery": "http://sample/first.rdf#i1536680377163170",
+                    "testCenter": "http://sample/first.rdf#i1536680377163171",
+                    "testTakers": [
+                      "http://sample/first.rdf#i1536680377163172",
+                      "http://sample/first.rdf#i1536680377163173"
+                    ]
+                  },
+                  "$ref": "#/components/schemas/Eligibility"
+                }
+              }
+            }
+          },
+          "404": {
```

Comment:
`400` errors are missing as it could return this error in case of non existing deliveries or test-centers.